.github/workflows/helm.yml

```diff
@@ -16,6 +16,16 @@ on:
         description: Repository {{ github.repository }} <owner>/<repository>
         default: ${{ github.repository }}
         required: false
+      force-publish:
+        description: Whether to enfore the publication of the chart. This chart version must be
+          and X.Y.Z-alpha{PR-NUMBER}.N for alpha releases.
+        type: boolean
+        required: false
+        default: false
+      pr-number:
+        description: PR number
+        type: number
+        required: false
```

Comment:
I'm not sure this will work as expected as the inputs are hard defined in the caller workflow. Have you tried it? You can set a caller workflow to use this branch with:
```yaml
jobs:
  test-generate-publish:
      uses: substra/substra-gha-workflows/.github/workflows/helm.yml@chore/allow-force-chart-publication
```

Reply:
Okay thanks, not didn't tried it, I'll do that :)